src/adzerk/boot_cljs/impl.clj

```diff
@@ -66,25 +67,34 @@
 (defn compile-cljs
   "Given a seq of directories containing CLJS source files and compiler options
   opts, compiles the CLJS to produce JS files."
-  [input-path {:keys [optimizations] :as opts}]
+  [input-path {:keys [optimizations asset-path] :as opts}]
   ;; So directories need to be passed to cljs compiler when compiling in dev
   ;; or there are stale namespace problems with tests. However, if compiling
   ;; with optimizations other than :none adding directories will break the
   ;; build and defeat tree shaking and :main option.
   (let [dirs (:directories pod/env)
-        ; Includes also some tmp-dirs passed to this pod, but shouldn't matter
+        ;; Includes also some tmp-dirs passed to this pod, but shouldn't matter
         source-paths (concat (:source-paths pod/env) (:resource-paths pod/env))
         warnings (atom [])
         handler (fn [warning-type env extra]
                   (when (warning-enabled? warning-type)
                     (when-let [s (ana/error-message warning-type extra)]
-                      (let [path (util/find-original-path source-paths dirs ana/*cljs-file*)]
-                        (butil/warn "WARNING: %s %s\n" s (when (:line env)
-                                                           (str "at line " (:line env) " " path)))
-                        (swap! warnings conj {:message s
-                                              :file path
-                                              :line (:line env)
-                                              :type warning-type})))))]
+                      (let [path (if (= (-> env :ns :name) 'cljs.core)
+                                   ;; AR - figwheel does this as well
+                                   (-> asset-path
```

Comment:
mmm, maybe you are right, I guess I was a bit overzealous, it might just be a bit confusing with multiple builds, while `:asset-path` would give us a path like: `main.out/cljs/core.cljs` or `foo.out/cljs/core.cljs` instead